doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -167,6 +168,9 @@ the fish shell (fish)                ``. util/quickstart/setchplenv.fish``
 the Bourne shell (sh)                ``. util/quickstart/setchplenv.sh``
 ==================================== ==========================================
 
+For the full-featured builds using these shells, simply remove
```

Comment:
For the full-featured, single-locale builds..."

Reply:
I'd rather not have to add a single-locale caveat each time we mention any of these settings, so adjusted the first paragraph to state the qualification early in order to avoid continuing to return to it (other than the bullet and section that explicitly point to the multi-locale settings documentation, which is not trivial / quickstart by nature):

> These instructions are designed to help users get started with a
> single-locale (shared-memory) implementation of Chapel from the source
> distribution of the release as quickly as possible.  If you are
> interested in taking the time to understand Chapel's configuration
> options, build process, and installation more thoroughly, please refer
> to :ref:`readme-chplenv` and :ref:`readme-building` instead.

(I'm also not sure that a multi-locale build is _always_ our preferred configuration.
e.g., if you're only running on shared-memory, we'd prefer you to use a single-locale
build to avoid the overheads that `--no-local` introduces).